index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="/css/styles.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Arturo Alvarez</title>
+</head>
+<body>
+	<header class="header">
+		<div class="logo">
+			My Logo
+		</div>
+		<div class="menu">
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
+				<path fill-rule="evenodd" d="M3 6.75A.75.75 0 013.75 6h16.5a.75.75 0 010 1.5H3.75A.75.75 0 013 6.75zM3 12a.75.75 0 01.75-.75h16.5a.75.75 0 010 1.5H3.75A.75.75 0 013 12zm0 5.25a.75.75 0 01.75-.75h16.5a.75.75 0 010 1.5H3.75a.75.75 0 01-.75-.75z" clip-rule="evenodd" />
+			</svg>
+		</div>
+	</header>
+	<main>
+		<section class="greet">
+			<div class="title">
+				<p>Welcome my portfolio</p>
+				<h1>Hi! I'm a web developer</h1>
```

Comment:
Please make sure to follow the Figma template exactly and remember to include your `name`.
Original:
<img width="303" alt="Screenshot 2023-03-23 at 17 01 05" src="https://user-images.githubusercontent.com/107192076/227390336-2ed9ac9d-3739-4504-81b5-3bb70ee3d9cb.png">

Yours:
<img width="327" alt="Screenshot 2023-03-23 at 16 59 53" src="https://user-images.githubusercontent.com/107192076/227390380-c2a05b0d-6ed5-4a6b-a0c4-c48cee4ce48e.png">